wdl/BAFTestChromosome.wdl

```diff
@@ -90,10 +91,14 @@ task BAFTest {
     }
   }
 
+  Int disk_gb = disk_gb_baseline + ceil(
```

Comment:
I didn't do any scaling regressions. I had (just a few) shards crash from out-of-disk errors, so I just added up the size of the input files. I think a few shards must wind up taking down a significant fraction of the baf_metrics file, because when I forgot to double it (for the 2x tabix) the shards still crashed. Incidentally, this is why I was removing those second tabix files in so many tasks. You didn't ask me to undo that for this task: should I assume it needs to be removed here too?

Reply:
Sorry, I spoke too soon. PETest and SRTest were having rare out-of-disk crashes, and I did have crashes in BAFTest as well. So I just implemented the same disk scaling in every test (PE, SR, RD, BAF). I just downloaded my monitoring logs and at least in my panel, BAFTest never used much of its disk. Given that it's 27 GiB on my 500 sample panel, I think it's reasonable leaving this in for safety sake. That said, it could probably could be removed for BAFTest (and RDTest as well).
It turns out BAFTest was crashing out of memory on one shard. Since I didn't have much to go on for scaling the memory usage of BAFTest, I just passed a runtime_attr override to my input json.

Reply:
We can revisit this once #51 goes in

Reply:
Also yes delete the `rm` statement below